Auxiliary/DirectXTexJPEG.cpp

```diff
@@ -0,0 +1,359 @@
+//--------------------------------------------------------------------------------------
+// File: DirectXTexJPEG.cpp
+//
+// DirectXTex Auxillary functions for using the JPEG(https://www.ijg.org) library
+//
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+//--------------------------------------------------------------------------------------
+
+#include "DirectXTexP.h"
+
+#include "DirectXTexJPEG.h"
+
+#if !defined(_BASETSD_H_)
+#define _BASETSD_H_
+#endif
+#include <cstdio>
+#include <filesystem>
+#include <jpeglib.h>
+#include <jerror.h>
+#include <memory>
+
+namespace DirectX
```

Comment:
You don't actually need a ``namespace DirectX`` in this file. All the hidden stuff should be in an anonymous namespace, which doesn't need to be embedded in the DirectX namespace.

Then to get things to build correctly:

1. add ``using namespace DirectX;` `
2. add to the 'free' functions that are public ``DirectX::`` 

I'm taking care of these changes in my reformat commit shortly.